morpheus/messages/message_meta.py

```diff
@@ -80,6 +82,13 @@ def __init__(self, df: pd.DataFrame) -> None:
         self._mutex = threading.RLock()
         self._df = df
 
+    @classmethod
+    def make_from_file(cls, filename: str):
+        if (cls._should_use_cpp()):
+            return cls._cpp_class.make_from_file(filename)
+        else:
+            return MessageMeta(read_file_to_df(filename, file_type=FileTypes.Auto, df_type="cudf"))
```

Comment:
A class method should call `cls(read_file_to_df(filename, file_type=FileTypes.Auto, df_type="cudf"))` to return the class we are calling this on. i.e. `UserMessageMeta.make_from_file()` would return a `MessageMeta` with the way it is.